src/main/java/com/expedia/www/haystack/metrics/appenders/logback/EmitToGraphiteLogbackAppender.java

```diff
@@ -103,7 +107,10 @@ public void setSubsystem(String subsystem) {
     @Override
     public void start() {
         super.start();
-        metricPublishing.start(new GraphiteConfigImpl(host, port, pollintervalseconds, queuesize, sendasrate));
+        // If disabled we do not create a publisher to graphite but error counts are still collected.
+        if(enabled) {
```

Comment:
do we need to add this check during stop too ?

Reply:
that's a good point. I dont think it matters for this specific implementation (it's using an atomic counter to check if the start was called) but it's probably better to add this check here as this may change in the future and we want to avoid coupling between the two classes.